jdbc/src/test/java/org/apache/zeppelin/jdbc/JDBCInterpreterTest.java

```diff
@@ -190,6 +190,27 @@ public void testSelectQuery() throws SQLException, IOException {
     assertEquals("ID\tNAME\na\ta_name\nb\tb_name\n", interpreterResult.message().get(0).getData());
   }
 
+  @Test
+  public void testColumnAliasQuery() throws IOException {
+    Properties properties = new Properties();
```

Comment:
Something must be wrong, this is not supposed to be in your commit. It is someone else's commit

Reply:
I checked the code I submitted, I did not modify this, is it caused by Git's bug?